src/ssd/ssd.c

```diff
@@ -340,9 +340,13 @@ ssd_enable_keybind_inhibit_indicator(struct ssd *ssd, bool enable)
 		return;
 	}
 
-	float *color = enable
-		? rc.theme->window_toggled_keybinds_color
-		: rc.theme->window_active_border_color;
+	float custom_color[4];
+	bool is_custom_color_available =
+		window_rules_get_custom_border_color(ssd->view, custom_color);
+
+	float *color = is_custom_color_available ? custom_color :
+		(enable ? rc.theme->window_toggled_keybinds_color
+				: rc.theme->window_active_border_color);
```

Comment:
I don't think the custom theme color should override the toggled-keybind color like this. Toggling keybinds is an ephemeral state, the indication of which is probably more important than honoring a color configured by a rule.

Reply:
To be honest, I haven't gone deep into the meaning of this part, I just noticed that it produces the bar on the top of the window, which is not colored. I will think how to resolve it (suggestions are also welcome!).